ERROR - 2022-12-20 02:33:35 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525204.702, 0.1615, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525204.702, 0.1615, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:33:35 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525204.702004, 0.1615, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:34:09 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525239.8262, 0.0379, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525239.8262, 0.0379, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:34:09 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525239.826197, 0.0379, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:39:22 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525552.8359, 0.0687, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525552.8359, 0.0687, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:39:22 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525552.835855, 0.0687, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:39:38 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525567.9805, 0.0586, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525567.9805, 0.0586, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:39:38 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525567.980528, 0.0586, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:40:26 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525616.5602, 0.0506, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525616.5602, 0.0506, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:40:26 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525616.560205, 0.0506, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:40:44 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525634.1656, 0.0494, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525634.1656, 0.0494, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:40:44 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525634.165557, 0.0494, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:40:55 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525645.1814, 0.0463, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525645.1814, 0.0463, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:40:55 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525645.181425, 0.0463, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:41:15 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525664.9146, 0.0653, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525664.9146, 0.0653, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:41:15 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525664.914602, 0.0653, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:41:32 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525682.8378, 0.0826, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525682.8378, 0.0826, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:41:32 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671525682.837817, 0.0826, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:47:29 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526038.9252, 0.147, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526038.9252, 0.147, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:47:29 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526038.925186, 0.147, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:48:27 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526096.866, 0.0943, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526096.866, 0.0943, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:48:27 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526096.865968, 0.0943, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:48:40 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526110.344, 0.0704, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526110.344, 0.0704, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:48:40 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526110.343965, 0.0704, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:57:45 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526655.2175, 0.1247, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526655.2175, 0.1247, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:57:45 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526655.217454, 0.1247, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-20 02:57:57 --> Error connecting to the database: mysqli_sql_exception: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('db4free.net', 'anitium', 'anitium123', 'anitium_db', 3306, '', 0)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#7 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#13 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526666.9114, 0.1199, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#20 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#21 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#22 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı in C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(NULL)
#5 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', NULL)
#6 C:\xampp\htdocs\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionCon...')
#7 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionCon...')
#8 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
#9 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
#10 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
#11 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
#12 C:\xampp\htdocs\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', Array)
#13 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
#14 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
#15 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526666.9114, 0.1199, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#16 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
#17 C:\xampp\htdocs\vendor\codeigniter4\framework\system\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), NULL)
#18 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('/', 'after')
#19 C:\xampp\htdocs\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#20 C:\xampp\htdocs\index.php(67): CodeIgniter\CodeIgniter->run()
#21 {main}
CRITICAL - 2022-12-20 02:57:57 --> Unable to connect to the database.
Main connection [MySQLi]: Bağlanılan uygun olarak belli bir süre içinde yanıt vermediğinden veya kurulan
bağlantı bağlanılan ana bilgisayar yanıt vermediğinden bir bağlantı kurulamadı
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(561): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `settings`
WHERE `context` IS NULL', [], false)
 3 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(163): CodeIgniter\Database\BaseBuilder->get()
 4 FCPATH\vendor\codeigniter4\settings\src\Handlers\DatabaseHandler.php(41): CodeIgniter\Settings\Handlers\DatabaseHandler->hydrate(null)
 5 FCPATH\vendor\codeigniter4\settings\src\Settings.php(59): CodeIgniter\Settings\Handlers\DatabaseHandler->has('Config\\Auth', 'sessionConfig', null)
 6 FCPATH\vendor\codeigniter4\settings\src\Helpers\setting_helper.php(21): CodeIgniter\Settings\Settings->get('Auth.sessionConfig')
 7 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(662): setting('Auth.sessionConfig')
 8 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(680): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionUserInfo()
 9 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(367): CodeIgniter\Shield\Authentication\Authenticators\Session->getSessionKey('id')
10 FCPATH\vendor\codeigniter4\shield\src\Authentication\Authenticators\Session.php(351): CodeIgniter\Shield\Authentication\Authenticators\Session->checkUserState()
11 FCPATH\vendor\codeigniter4\shield\src\Auth.php(160): CodeIgniter\Shield\Authentication\Authenticators\Session->loggedIn()
12 FCPATH\vendor\codeigniter4\shield\src\Collectors\Auth.php(67): CodeIgniter\Shield\Auth->__call('loggedIn', [])
13 SYSTEMPATH\Debug\Toolbar\Collectors\BaseCollector.php(222): CodeIgniter\Shield\Collectors\Auth->display()
14 SYSTEMPATH\Debug\Toolbar.php(93): CodeIgniter\Debug\Toolbar\Collectors\BaseCollector->getAsArray()
15 SYSTEMPATH\Debug\Toolbar.php(374): CodeIgniter\Debug\Toolbar->run(1671526666.911414, 0.1199, Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
16 SYSTEMPATH\Filters\DebugToolbar.php(40): CodeIgniter\Debug\Toolbar->prepare(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response))
17 SYSTEMPATH\Filters\Filters.php(196): CodeIgniter\Filters\DebugToolbar->after(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), null)
18 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\Filters\Filters->run('C:\\', 'after')
19 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
20 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
